cmd/metrics-v3-api-bucket-replication.go

```diff
@@ -0,0 +1,155 @@
+	bucketL                                        = "bucket"
+	operationL                                     = "operation"
+	targetArnL                                     = "targetArn"
+)
+
+var (
+	bucketReplLastHrFailedBytesMD = NewGaugeMD(bucketReplLastHrFailedBytes,
+		"Total number of bytes failed at least once to replicate in the last hour on a bucket",
+		bucketL)
+	bucketReplLastHrFailedCountMD = NewGaugeMD(bucketReplLastHrFailedCount,
+		"Total number of objects which failed replication in the last hour on a bucket",
+		bucketL)
+	bucketReplLastMinFailedBytesMD = NewGaugeMD(bucketReplLastMinFailedBytes,
+		"Total number of bytes failed at least once to replicate in the last full minute on a bucket",
+		bucketL)
+	bucketReplLastMinFailedCountMD = NewGaugeMD(bucketReplLastMinFailedCount,
+		"Total number of objects which failed replication in the last full minute on a bucket",
+		bucketL)
+	bucketReplLatencyMsMD = NewGaugeMD(bucketReplLatencyMs,
+		"Replication latency on a bucket in milliseconds",
+		bucketL, operationL, rangeL, targetArnL)
+	bucketReplProxiedDeleteTaggingRequestsTotalMD = NewCounterMD(bucketReplProxiedDeleteTaggingRequestsTotal,
+		"Number of DELETE tagging requests proxied to replication target",
+		bucketL)
+	bucketReplProxiedGetRequestsFailuresMD = NewCounterMD(bucketReplProxiedGetRequestsFailures,
+		"Number of failures in GET requests proxied to replication target",
+		bucketL)
+	bucketReplProxiedGetRequestsTotalMD = NewCounterMD(bucketReplProxiedGetRequestsTotal,
+		"Number of GET requests proxied to replication target",
+		bucketL)
+	bucketReplProxiedGetTaggingRequestsFailuresMD = NewCounterMD(bucketReplProxiedGetTaggingRequestsFailures,
+		"Number of failures in GET tagging requests proxied to replication target",
+		bucketL)
+	bucketReplProxiedGetTaggingRequestsTotalMD = NewCounterMD(bucketReplProxiedGetTaggingRequestsTotal,
+		"Number of GET tagging requests proxied to replication target",
+		bucketL)
+	bucketReplProxiedHeadRequestsFailuresMD = NewCounterMD(bucketReplProxiedHeadRequestsFailures,
+		"Number of failures in HEAD requests proxied to replication target",
+		bucketL)
+	bucketReplProxiedHeadRequestsTotalMD = NewCounterMD(bucketReplProxiedHeadRequestsTotal,
+		"Number of HEAD requests proxied to replication target",
+		bucketL)
+	bucketReplProxiedPutTaggingRequestsFailuresMD = NewCounterMD(bucketReplProxiedPutTaggingRequestsFailures,
+		"Number of failures in PUT tagging requests proxied to replication target",
+		bucketL)
+	bucketReplProxiedPutTaggingRequestsTotalMD = NewCounterMD(bucketReplProxiedPutTaggingRequestsTotal,
+		"Number of PUT tagging requests proxied to replication target",
+		bucketL)
+	bucketReplSentBytesMD = NewCounterMD(bucketReplSentBytes,
+		"Total number of bytes replicated to the target",
+		bucketL)
+	bucketReplSentCountMD = NewCounterMD(bucketReplSentCount,
+		"Total number of bytes replicated to this bucket from another source bucket",
```

Comment:
The two description somehow look confusing to me.
`sent_bytes` says bytes sent to the target, whereas `sent_count` says bytes replicated to this bucket from another source.
cc @poornas

Reply:
sent_count is total number of object versions from this bucket to target

Reply:
Thanks, fixed.